site/en/integrations/integrate_with_sentencetransformers.md

```diff
@@ -6,222 +6,176 @@ title: Movie Search Using Milvus and SentenceTransformers
-    ```
+```python
+connections.connect(uri="./sentence_transformers_example.db")
+```
 
-3. Create the collection that holds the id, title of the movie, and the embeddings of the plot text.
+2. Create the data schema. This specifies the fields that comprise an element including the dimension of the vector embedding.
 
-    ```python
-    from pymilvus import FieldSchema, CollectionSchema, DataType, Collection
+```python
+fields = [
+    FieldSchema(name='id', dtype=DataType.INT64, is_primary=True, auto_id=True),
+    FieldSchema(name='title', dtype=DataType.VARCHAR, max_length=256),
+    FieldSchema(name='embedding', dtype=DataType.FLOAT_VECTOR, dim=embedding_dim)
+]
+
+schema = CollectionSchema(fields=fields, enable_dynamic_field=False)
+collection = Collection(name=collection_name, schema=schema)
+```
 
+3. Define the vector search indexing algorithm. Milvus Lite implements brute force search and HNSW, whereas Milvus Standalone and Milvus Distributed implement a wide variety of methods. For this scale of data, the naive brute force search suffices.
 
-    # Create collection which includes the id, title, and embedding.
-    fields = [
-        FieldSchema(name='id', dtype=DataType.INT64, is_primary=True, auto_id=True),
-        FieldSchema(name='title', dtype=DataType.VARCHAR, max_length=200),  # VARCHARS need a maximum length, so for this example they are set to 200 characters
-        FieldSchema(name='embedding', dtype=DataType.FLOAT_VECTOR, dim=DIMENSION)
-    ]
-    schema = CollectionSchema(fields=fields)
-    collection = Collection(name=COLLECTION_NAME, schema=schema)
-    ```
-
-4. Create an index on the newly created collection and load it into memory.
-
-    ```python
-    # Create an IVF_FLAT index for collection.
-    index_params = {
-        'metric_type':'L2',
-        'index_type':"IVF_FLAT",
-        'params':{'nlist': 1536}
+```python
+params = {
+    'index_type':"FLAT",
+    'metric_type': "IP"
     }
-    collection.create_index(field_name="embedding", index_params=index_params)
-    collection.load()
-    ```
-
-Once these steps are done the collection is ready to be inserted into and searched. Any data added will be indexed automatically and be available to search immediately. If the data is very fresh, the search might be slower as brute force searching will be used on data that is still in process of getting indexed.
 
-## Inserting the data
+collection.create_index(
+    'embedding',
+    params
+)
+```
 
-For this example, we are going to use the SentenceTransformers miniLM model to create embeddings of the plot text. This model returns 384-dim embeddings.
+Once these steps are done, we are ready to insert data into the collection and perform a search. Any data added will be indexed automatically and be available to search immediately. If the data is very fresh, the search might be slower as brute force searching will be used on data that is still in process of getting indexed.
```

Comment:
this is a weird statement. the search is brute search (FLAT) anyways. This only makes a difference for other index types, say HNSW, where sealed segment has HNSW, while growing segment (data just inserted and not auto flushed yet) are slower (for HNSW the growing segment uses something like IVF but not exactly, but it's not FLAT, and it's faster than FLAT)